samples/bookinfo/src/productpage/Dockerfile

```diff
@@ -23,6 +23,7 @@ RUN pip install --no-cache-dir -r test-requirements.txt
 COPY productpage.py /opt/microservices/
 COPY tests/unit/* /opt/microservices/
 COPY templates /opt/microservices/templates
+COPY static /opt/microservices/static
```

Comment:
Instead of storing these files (Bootstrap, jQuery, etc.), can these be fetched while building the Docker image?

Reply:
@venilnoronha any reason why fetching them on build is better? I'm thinking that might be more error prone.

Reply:
It's actually hard to review minified files, so I'd prefer if we pull this from a trusted source while building the image.

Reply:
I grab that files down from original link provided by CDN. You can check checksum for minified file. If you are uncomfortable with that, it can be changed to be fetched during building. If so, please leave a comment so I can close this PR (and have another one)

Reply:
Alright, let's keep this as is.